aries_vcx/src/protocols/trustping/mod.rs

```diff
@@ -1,45 +1,24 @@
-use std::clone::Clone;
-use std::future::Future;
 
-use indy_sys::WalletHandle;
-
-use crate::did_doc::DidDoc;
-use crate::error::prelude::*;
-
-use crate::messages::a2a::A2AMessage;
+use crate::messages::a2a::MessageId;
 use crate::messages::trust_ping::ping::Ping;
 use crate::messages::trust_ping::ping_response::PingResponse;
+use crate::utils::uuid;
+
+pub fn build_ping(request_response: bool, comment: Option<String>) -> Ping {
+    // todo : Remove different Default implementation for MessageId in tests, then we can remove this override
```

Comment:
Next time, no need to put a comment everywhere applicable. Just a comment in the PR description / an issue would be better.